source/defclass-star.lisp

```diff
@@ -455,7 +480,7 @@ New class options (defaults are NIL unless specified):
 - `:type-inference' (default: `default-type-inference').
 
   Function that returns the type of a slot definition (with an initform).  It
-takes 3 arguments:
+  takes 3 arguments:
```

Comment:
Some indentation is off?

Reply:
It was wrong before I think.

Reply:
Ah, I see now. The format is not exactly cool (I'd personally do something like Shinmera-styled formatting), but we can live with it :)